test/Spec.hs

```diff
@@ -1,10 +1,58 @@
+import Data.List
+import Data.Function
 import Boids
 import Test.HUnit
-import Test.HUnit.Approx
 import System.Exit
 
+testIsWithinView :: Test
+testIsWithinView  = TestCase (assertBool "Seeing a boid behind me" (not $ isWithinView boid1 boid2))
```

Comment:
Ah, okay; so the message is written such that if test fails, it's obvious what's wrong. Got it!